openshift/template.yaml

```diff
@@ -111,6 +111,11 @@ objects:
               configMapKeyRef:
                 name: launcher
                 key: launcher.tracker.segment.token
+          - name: LAUNCHER_SENTRY_DSN
+            valueFrom:
+              configMapKeyRef:
+                name: launcher
+                key: launcher.sentry.dsn
```

Comment:
I think this needs an indication that it belongs to frontend, and also needs to be `optional: true`

Reply:
They all seem to be optional and for the frontend..